metadata.json

```diff
@@ -7,6 +7,35 @@
   "source": "git://github.com/Ericsson/puppet-module-cron.git",
   "project_page": "https://github.com/Ericsson/puppet-module-cron",
   "issues_url": "https://github.com/Ericsson/puppet-module-cron/issues",
+  "operatingsystem_support": [
+    {
+      "operatingsystem": "Debian",
+      "operatingsystemrelease": [
+        "6"
```

Comment:
This should be added to the README and to the spec tests in the platforms hash at https://github.com/Ericsson/puppet-module-cron/blob/master/spec/classes/init_spec.rb#L46

Would also be good to have someone else verify it works on that platform.